src/ibex_bluesky_core/callbacks/_fitting.py

```diff
@@ -22,11 +23,94 @@
     get_default_output_path,
     get_instrument,
 )
-from ibex_bluesky_core.callbacks.fitting import LiveFit
+from ibex_bluesky_core.fitting import FitMethod
 
 logger = logging.getLogger(__name__)
 
```

Comment:
Add an `__all__` for this module to make it clear what APIs should be imported from here?